cpp/autosar/src/rules/A8-4-7/TriviallyCopyableSmallType.qll

```diff
@@ -12,7 +12,7 @@ int wordSize() { result = max(VoidPointerType v | | v.getSize()) }
  * Converts bytes to words
  */
 bindingset[bytes]
-int bytesToWords(int bytes) { result = bytes / wordSize() }
+float bytesToWords(float bytes) { result = bytes / wordSize() }
```

Comment:
Since there are some nuances to equality testing in floats happening in `bytesToWords(size) <= 2` I suggest to change this to:

```suggestion
int minWordsRequiredToRepresentBytes(int bytes) { result = (1.0*bytes / wordSize()).ceil() }
```